resume.html

```diff
@@ -37,22 +40,29 @@
 </head>
 
 <body>
-	<h2>
-		<img id="self-photo" src="assets\images\self-photo.png" alt="This is a photo ID style self 
-portrait of Michael."> MICHAEL ALEXANDER DENMAN
-	</h2>
-	<address>
-		<a id="phone-number" href="tel:+13252772340">Call me</a> --
-		<a id="email-link"
-			href="mailto:denmanmd@gmail.com?subject=Resume Response&body=In response to madenman.me/resume">Email Me</a>
-		--
-		<a id="linkedin-link" href='https://www.linkedin.com/in/michael-denman/'> Check out my LinkedIn</a><br>
-	</address>
-	United States Citizen, Languages: Spanish & English<br>
-	<a id="sec+-link" href="https://www.youracclaim.com/badges/1c4d9cb9-b9da-4602-b60a-48be5a33ad94">Security+</a> &
-	<a id="casp+-link" href="https://www.youracclaim.com/badges/b698b10a-25f6-4f20-9d5a-5455360fffba">CASP+</a>
-	Certified, Job Availability Date: 23 Sept 2019<br>
-
+	<div style="background-color: #000000; max-width: 600px;">
+		<div style="align-content:flex-start;">
+			<img id="self-photo" src="assets\images\self-photo.png" alt="Self-photo" style="vertical-align: top; float:left;">
+		</div>
+		<div style="float:inline-end;">
+			<h2>MICHAEL ALEXANDER DENMAN</h2>
+			<address>
+				<a id="phone-number" href="tel:+13252772340">Call Me</a>
+				--
+				<a id="email-link"
+					href="mailto:denmanmd@gmail.com?subject=Resume Response&body=In response to madenman.me/resume">Email
+					Me</a>
+				--
+				<a id="linkedin-link" href='https://www.linkedin.com/in/michael-denman/'> Check out my LinkedIn</a><br>
+			</address>
+			United States Citizen, Languages: Spanish & English<br>
+			<a id="sec+-link"
+				href="https://www.youracclaim.com/badges/1c4d9cb9-b9da-4602-b60a-48be5a33ad94">Security+</a> &
+			<a id="casp+-link" href="https://www.youracclaim.com/badges/b698b10a-25f6-4f20-9d5a-5455360fffba">CASP+</a>
+			Certified, Job Availability Date: 23 Sept 2019<br>
+		</div>
+
+	</div>
```

Comment:
Trying to get all this content centered and in it's own box

Reply:
@denmanma It's time to play a game! https://flexboxfroggy.com/